index.html

```diff
@@ -0,0 +1,114 @@
+    </nav>
+    <section class="heading">
+      <p class="supporting-heading">Discover the Filmmaking Magic</p>
+      <h1 class="main-heading">Explore the wonders of the Wizarding World</h1>
+      <div class="description">
+        <p class="description-text">The Studio Tour offers visitors the unique opportunity to explore two soundstages and a backlot filled with original sets, animatronic creatures and breathtaking special effects.</p>
+      </div>
+      <div class="booking-section">
+        <button id="book-now-btn">Book Now</button>
+        <p class="booking-description">MUST BE BOOKED IN ADVANCE</p>
+      </div>
+    </section>
+  </header>
+  <main>
+    <section id="experience" class="experience-section">
+      <h3 class="experience-heading">The Experience</h3>
+      <div id="experience-heading-under"></div>
+      <ul class="experiences-list">
+        <li id="experience-1" class="experience">
+          <div class="experience-image-1"></div>
+          <h4 class="experience-main">Sets</h4>
+          <p class="experience-info">Step into the Great Hall, Forbidden Forest and Diagon Alley</p>
+        </li>
+        <li id="experience-2" class="experience">
+          <div class="experience-image-2"></div>
+          <h4 class="experience-main">Props</h4>
+          <p class="experience-info">Explore the collection of beautifully crafted iconic props</p>
+        </li>
+        <li id="experience-3" class="experience">
+          <div class="experience-image-3"></div>
+          <h4 class="experience-main">Costumes</h4>
+          <p class="experience-info">See up close the intricate detail hidden in every costume</p>
+        </li>
+        <li id="experience-4" class="experience">
+          <div class="experience-image-4"></div>
+          <h4 class="experience-main">Effects</h4>
+          <p class="experience-info">Discover how the special effects teams made Harry and his friends fly</p>
+        </li>
+        <li id="experience-5" class="experience">
+          <div class="experience-image-5"></div>
+          <h4 class="experience-main">Creatures</h4>
+          <p class="experience-info">Learn how magical creatures were brought to life on screen</p>
+        </li>
+        <li id="experience-6" class="experience">
+          <div class="experience-image-6"></div>
+          <h4 class="experience-main">Art</h4>
+          <p class="experience-info">From technical drawings to scale models, learn how sets were created</p>
+        </li>
+      </ul>
+      <button id="find-out-btn"><a href="" id="find-out-btn-link">Find out more</a></button>
+      <a href="" id="find-out-btn-link-lg">Find out more</a>
+    </section>
+    <section id="featured" class="featured-section">
+      <h3 class="featured-heading">What's On</h3>
+      <div id="featured-heading-under"></div>
+      <ul class="featured"></ul>
+    </section>
+    <section id="awards" class="awards-section">
+      <h3 class="awards-heading">Award-Winning Tour</h3>
```

Comment:
- [ ] Kindly adjust the background color of your `awards`  section to match the original design

![image](https://user-images.githubusercontent.com/98453979/204054181-66503a40-1586-4ded-aaed-7a510dac51eb.png)

### Checkout the original design
![image](https://user-images.githubusercontent.com/98453979/204054211-8a75f1e8-1468-4b4d-b9fc-e727349da96d.png)